### DISCUSSION TITLE
Acil

### DISCUSSION THREAD

**Discussion by Ylmz038** [cedricp/ddt4all#929] [Q&A]
Hello, my vehicle is Renault Scenic 3. I do not know what I am doing while I am tampering with the ABS and ESP settings. The ABS light is constantly on. How can I turn it off and the hill start assist is not working, please help me.

Uploading VID_20231003_171717.mp4…

**Comment by AndyIreland** [comment]
> Hello, my vehicle is Renault Scenic 3. I do not know what I am doing while I am tampering with the ABS and ESP settings. The ABS light is constantly on. How can I turn it off and the hill start assist is not working, please help me.
> 
> Uploading VID_20231003_171717.mp4…

hi, either enable ABS/ESP back again or extinguish the light by going into your 'clustre' settings and choose 'no ABS' and/or ' no eps'

**Comment by Ylmz038** [comment]
Nasıl yapacağım bunu örnek bı fotoğraf var mi

**Comment by AndyIreland** [comment]
> How can I do this? is there an example photo?

Well - depending on what cluster you have when you do a search with the DDT4all magnifying glass to find your ECU's fitted on your car - here is an example:
![image](https://github.com/cedricp/ddt4all/assets/49172217/2ba32f01-f2b1-45e1-b000-16566265441c)

Sans means 'without' and Avec means 'With' 

this is how you can extinguish the warning lights from your dashboard

**Comment by Ylmz038** [comment]
Çok teşekkür ederim söylediğinizi deneyeceğim
